modules/administration/pages/auth-methods-pam.adoc

```diff
@@ -1,44 +1,32 @@
 [[auth-methods-pam]]
 = Authentication With PAM
 
-{productname} supports network-based authentication systems using pluggable authentication modules (PAM).
+{productname} supports network-based authentication systems using pluggable authentication modules (PAM) using SSSD.
 PAM is a suite of libraries that allows you to integrate {productname} with a centralized authentication mechanism, eliminating the need to remember multiple passwords.
-{productname} supports LDAP, Kerberos, and other network-based authentication systems using PAM.
+{productname} supports LDAP, Kerberos, and other network-based authentication.
 
 
 
-.Procedure: Enabling PAM
-. Create a PAM service file at [path]``/etc/pam.d/susemanager``.
-  Filename must be in lower case and readable by ``tomcat`` user.
-  This file is used by {productname} to load the correct PAM configuration files:
+.Procedure: Configure SSSD
+. In the {productname} {webui}, navigate to menu:Users[Create User] and enable a new or existing user to authenticate with PAM.
+. Check the [guimenu]``Pluggable Authentication Modules (PAM)`` checkbox.
+  It is below the password and password confirmation fields.
+. Configure SSSD in the container. Connect inside the container using:
 +
 ----
-#%PAM-1.0
-auth     include        common-auth
-account  include        common-account
-password include        common-password
-session  include        common-session
+mgrctl term
 ----
-. On the {productname} Server, at the command prompt, as root, add the ``sss`` PAM module:
+Then change the file
 +
 ----
-pam-config -a --sss
+/etc/sssd/sssd.conf
```

Comment:
sorry @keichwa I was confused since I saw the comment after a long time. Actually `pam-config -a --sss` it's not required

Reply:
as you can see from this commit, it's already performed during the image build phase https://github.com/uyuni-project/uyuni/pull/8537/ . The file are created inside `/etc/pam.d` folder and since this folder it's not persisted, the used should not re-run it. Sorry for the confusion

Reply:
Thanks for clarification!